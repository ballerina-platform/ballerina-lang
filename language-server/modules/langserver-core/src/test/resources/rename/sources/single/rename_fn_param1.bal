# Calculates the euclidean distance between two float vectors.
#
# + vector1 - float vector 1
# + vector2 - float vector 2
# + return - the euclidean distance between the given two vectors
function euclideanDistance(float[] vector1, float[] vector2) returns float {
    return vector1[0];
}
