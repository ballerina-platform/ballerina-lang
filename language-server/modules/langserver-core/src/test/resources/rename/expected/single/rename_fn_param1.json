{
  "source": {
    "file": "rename_fn_param1.bal"
  },
  "position": {
    "line": 6,
    "character": 14
  },
  "prepareRename": {
    "valid": true
  },
  "result": {
    "changes": {
      "rename_fn_param1.bal": [
        {
          "range": {
            "start": {
              "line": 5,
              "character": 35
            },
            "end": {
              "line": 5,
              "character": 42
            }
          },
          "newText": "v1"
        },
        {
          "range": {
            "start": {
              "line": 6,
              "character": 11
            },
            "end": {
              "line": 6,
              "character": 18
            }
          },
          "newText": "v1"
        },
        {
          "range": {
            "start": {
              "line": 2,
              "character": 4
            },
            "end": {
              "line": 2,
              "character": 11
            }
          },
          "newText": "v1"
        }
      ]
    }
  }
}
